src/actions/index.js

```diff
@@ -159,6 +159,12 @@ export const toggleGamePause = () => {
   };
 };
 
+export const pauseGameAction = () => {
```

Comment:
Let's drop the 'Action' suffix to match the other actions (lol).

I'm also thinking we should put a `if (paused)` guard on this? See toggleGamePause above.

Reply:
Is the naming better or worse now? 😆  Specifically this commit: [23e0985603ae8fde99fe828437b9e9eacd09d3b6](https://github.com/BattlesnakeOfficial/board/pull/49/commits/23e0985603ae8fde99fe828437b9e9eacd09d3b6)

I changed the existing `pauseGame` to `receivePauseGame` kinda like some of the more websockey events. Not sure its the best, but it let me use `pauseGame` for the actiony method!

Reply:
Yeah it's kinda worse eh? Hrmm... I didn't realize there was a name collision 🤦

I'd vote to go back to what you had... (sorrrryyyyyy)

Reply:
No problem!

Will revert to the old naming